modules/nw-controlling-dns-pod-placement.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * networking/dns-operator.adoc
+
+[id="nw-controlling-dns-pod-placement_{context}"]
+= Controlling DNS pod placement
+
+The DNS Operator has two daemon sets, one for CoreDNS and one for managing `/etc/hosts`. The daemon set for `etc/hosts` must run on every node host and include an entry for the Cluster Image Registry to support pulling images. Security policies can prohibit communication between pairs of nodes, which prevents the daemon set for CoreDNS from running on every node.
+
+As a cluster administrator, you can use a custom node selector to configure the daemon set for CoreDNS to run or not run on certain nodes.
+
+
+.Prerequisites
+
+* You installed the `oc` CLI.
+* You are logged in to the cluster with a user with `cluster-admin` privileges.
+
+.Procedure
+
+* Configure the `spec.nodePlacement.nodeSelector` API field to prevent communication between certain nodes.
+
+. Modify the DNS Operator object named default:
++
+[source, terminal]
+----
+$ oc edit dns.operator default
```

Comment:
Should this be `dns.operator/default`? (In both commands.)

Reply:
yes, please follow the `RESOURCE/NAME` style.

Reply:
Thank you! Updated.